src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -51,7 +58,7 @@ public async Task<ActionResult> UploadAutoReview([FromForm] IFormFile file, stri
                     if (apiRevision != null)
                     {
                         apiRevision = await _apiRevisionsManager.UpdateRevisionMetadataAsync(apiRevision, packageVersion ?? codeFile.PackageVersion, label, setReleaseTag);
-                        var reviewUrl = $"{this.Request.Scheme}://{this.Request.Host}/Assemblies/Review/{apiRevision.ReviewId}?revisionId={apiRevision.Id}";
+                        var reviewUrl = ManagerHelpers.ResolveReviewUrl(reviewId: apiRevision.ReviewId, apiRevisionId: apiRevision.Id, language: apiRevision.Language, configuration: _configuration, languageServices: _languageServices);
```

Comment:
what was affected by this?

Reply:
It just shows the old URI on devOps. But redirect happens when it is clicked.